src/neptune/common/envs.py

```diff
@@ -18,6 +18,7 @@
 API_TOKEN_ENV_NAME = "NEPTUNE_API_TOKEN"
 
 NEPTUNE_RETRIES_TIMEOUT_ENV = "NEPTUNE_RETRIES_TIMEOUT"
+NEPTUNE_SYNC_AFTER_STOP_TIMEOUT_ENV = "NEPTUNE_SYNC_AFTER_STOP_TIMEOUT"
```

Comment:
There is something like `neptune.internal.constants` (non user-exposed constants) + Let's do not extend the `common` if legacy client is not using it :wink: